src/frontend/components/Summary/index.js

```diff
@@ -44,34 +46,38 @@ export default class Summary extends Component {
     const active = getActive();
     const busy = getBusy();
     const onDuty = getOnDuty();
-
+    /* eslint-disable */
```

Comment:
🔴

Reply:
This has to be here to disable the warning of the setInnerHtml. It's ok

Reply:
Create a function that sets the html like `renderTree` and just disable the specific rule there, otherwise you can disable a lot of other linter warnings.

Reply:
done